### ISSUE TITLE
[Bug]: postgres use_jsonb

### ISSUE THREAD

**Issue by bogdanalexandrutreica** [run-llama/llama_index#8961]
### Bug Description

Postgresq vectore store is not working after use_jsonb PR

Locally I solved the problem by adding

use_jsonb=use_jsonb

on line 188 in postgres.py

### Version

0.9.1

### Steps to Reproduce

Create a vector store with pgvector

### Relevant Logs/Tracbacks

```shell
https://github.com/run-llama/llama_index/pull/8910

as it is mentioned in this PR by sunnybak


ValidationError: 1 validation error for PGVectorStore
use_jsonb
field required (type=value_error.missing)
```

**Comment by tolexy** [comment]
I commented on the merge, but wanted to also raise this on the issue itself. 

This is still an issue when using 0.9.2. The same error is appearing. Using llama-index==0.9.2 in Google Colab. Code throws error when getting to the PGVectorStore process. All worked fine a few days ago. Rolling back to previous versions of llama-index did not solve the issue either. I see the "fix" was merged, but it does not seem fixed. Any suggestions?

url = make_url(connection_string)
vector_store = PGVectorStore.from_params(
database=db_name,
host=url.host,
password=url.password,
port=url.port,
user=url.username,
table_name=table_name,
embed_dim=1536,
)

/usr/local/lib/python3.10/dist-packages/pydantic/main.cpython-310-x86_64-linux-gnu.so in pydantic.main.BaseModel.init()
ValidationError: 1 validation error for PGVectorStore
use_jsonb
field required (type=value_error.missing)
